src/lib/components/Login/OnaOauthLogin/OnaOauthLogin.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { useOAuthLogin } from '@onaio/gatekeeper/dist/components/login';
+import { AuthorizationGrantType } from '@onaio/gatekeeper';
+
+class OnaOauthLogin extends Component {
+  constructor(props) {
+    super(props);
+    const clientID = this.props.clientID
```

Comment:
Let's use the `deconstructing` syntax here pls `const { clientID } = this.props;` 🙏